src/commands/cmd_bulk_insert.c

```diff
@@ -127,8 +127,10 @@ static void _MGraph_BulkInsert(void *args) {
 	// free retained strings
 	for(int i = 0; i < argc; i++) RedisModule_FreeString(ctx, argv[i]);
 
+	QueryCtx_Free();
```

Comment:
Please explain why this `QueryCtx_Free` call is necessary ?

Reply:
When we create a `GraphContext`, it allocates and sets itself in a `QueryCtx`.